src/core/public/integrations/moment/moment_service.test.ts

```diff
@@ -47,6 +47,30 @@ describe('MomentService', () => {
     expect(momentMock.updateLocale).toHaveBeenCalledWith('default-locale', { week: { dow: 0 } });
   });
 
+  it('uses the default timezone when a zone is not defined', async () => {
+    const tz$ = new BehaviorSubject('tz4');
```

Comment:
This test might be a bit more obvious if the tz name was something like `tz-does-not-exist`